crates/blockchain-tree/benches/bench.rs

```diff
@@ -0,0 +1,53 @@
+#![allow(missing_docs)]
+
+use alloy_consensus::Header;
+use alloy_primitives::BlockHash;
+use criterion::{self, black_box, criterion_group, criterion_main, Criterion};
+use reth_blockchain_tree::BlockBuffer;
+use reth_primitives::{BlockBody, SealedBlock, SealedBlockWithSenders, SealedHeader};
+
+pub fn create_block(_number: u64, _parent: BlockHash) -> SealedBlockWithSenders {
+    let _rng = rand::thread_rng();
+
+    let header = Header::default();
+    let s_header = SealedHeader::new(header, BlockHash::random());
+
+    let a = BlockBody::default();
+    let sealed_block = SealedBlock::new(s_header, a);
+    SealedBlockWithSenders::new(sealed_block, vec![]).unwrap()
+}
```

Comment:
this benchmark does not actually test for large removals in `remove_old_blocks`, since the block number is unused, so there is no guarantee the blocks are linked together.

Reply:
still mostly same as before